docker-compose.yml

```diff
@@ -1,47 +1,25 @@
-
```

Comment:
Why does this PR remove the license header? Also, any new files shiould include the license header.